news/7146.feature

```diff
@@ -0,0 +1 @@
+This change adds CA information to ``pip debug``.
```

Comment:
May be clearer if worded like "Add CA certificate information to \`\`pip debug\`\`."